components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/HumanTaskServer.java

```diff
@@ -163,10 +170,10 @@ private void initDataSource() throws HumanTaskServerException {
         //TODO - need to handle the external transaction managers.
         database.setTransactionManager(tnxManager);
         try {
-	        database.start();
+            database.start();
         } catch (Exception e) {
             String errMsg = "Humantask Database Initialization failed.";
-            log.error(errMsg);
+            log.error(errMsg, e);
```

Comment:
This log can be removed as we are throwing the error. Otherwise, this will cause duplicate logs.